companies/iamfy.json

```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "iamfy",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Project J UG",
+    "runs": [
+        "Fy!",
+        "Project J Ltd."
+    ],
+    "address": "Glogauer Straße 5\nHinterhof Rechts\n4. OG\n10999 Berlin\nDeutschland",
+    "email": "dataprotection@iamfy.co",
+    "web": "https://www.iamfy.co/",
+    "sources": [
+        "https://www.iamfy.co/privacy"
+    ],
+    "request-language": "en",
```

Comment:
Sidenote: We need to figure out a way way to say that English and German are possible here.

Reply:
I agree that we might want to implement that but I don't think it applies in this particular case. 

Even though the company is from Germany, from what I can see, the website is only offered in English.